pyproject.toml

```
@@ -11,59 +11,31 @@ documentation = "https://kuma.readthedocs.io"
 [tool.poetry.dependencies]
 python = "^3.8"
 
-babel = "^2.8.0"
-boto3 = "^1.10.37"
-botocore = "^1.14.17"
 celery = "^4.3.0"
 dj-database-url = "^0.5.0"
 dj-email-url = "^1"
-django = "^2"
-django-allauth = "^0.41.0"
+django = "^3"
 django-decorator-include = "^3.0"
 django-extensions = "^2.2.6"
-django-jinja = "^2.7.0"
-django-mysql = "^3.3.0"
-django-pipeline = "^1.7.0"
 django-ratelimit = "2.0.0"
 django-redis = "^5.0.0" # (Django cache backend)
-django-statici18n = "^1.9.0"
-django-storages = "^1.9.1"
-django-taggit = "^0.24.0"
 django-waffle = "^0.20.0"
-djangorestframework = "^3.11.2"
 elasticsearch = "^7.13.1"
 elasticsearch-dsl = "^7.3.0"
-enmerkar = "^0.7.1" # Maintained fork of django-babel
-google-api-python-client = "^2.10.0"
 gunicorn = "^20.1.0"
-html5lib = "1.0.1"
-httplib2 = "^0.17.0"
-jinja2 = "^3.0.1"
-lxml = "^4.6.3"
-mysqlclient = "^1.4.6" # (Django database driver)
 newrelic = "^5.14.0.142"
```

We need to upgrade `newrelic` to the latest so it stays in sync with Django (see https://github.com/newrelic/newrelic-python-agent/commit/2db7dc607cf8717c212f2839e426a9d8d820e80d). I've already tested this on stage, and it fixes the 500 errors that we get when we should be returning 404's.
```suggestion
newrelic = "^6.4.4.161"
```